changelog/changelog_3.0.0-4.0.0.txt

```diff
@@ -1,3 +1,14 @@
+11.10.2024
+Description
+    - Add methods in function block to add/remove nestead fb
+
+Required integration changes:
+    - Breaks binary compatibility
```

Comment:
This needs more details explaining how this impacts function block writing. Basically all function blocks using nested function blocks must be rewritten to suit this PR.